src/main/java/seedu/letsgethired/model/application/exceptions/ApplicationNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package seedu.letsgethired.model.application.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified intern application.
+ */
+public class ApplicationNotFoundException extends RuntimeException {}
```

Comment:
Might be better to name this `InternApplicationNotFoundException`, because the current name might be confused with our LetsGetHired program being missing or something.

Reply:
Good find Zhi Sheng! Thanks :D